torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -313,11 +315,13 @@ void ProfilingGraphExecutorImpl::runProfilingOptimizations(
     runNoGradOptimizations(copy);
   }
   EliminateDeadCode(copy);
+  ClearProfilingInformation(copy);
   GRAPH_DUMP("After runProfilingOptimizations:", copy);
 }
 
 void ProfilingGraphExecutorImpl::runProfilingInsensitiveOptimizations(
     std::shared_ptr<Graph>& graph) {
+  Inline(*graph);
```

Comment:
Nit: might be worth adding a comment why we need it. I assume it's because we're now planning to generate function calls for fallbacks (if I didn't know that I'd assume this pass was useless)?

Reply:
We weren't running inlining in the forward graph of differentiable subgraphs at all

Reply:
Previously we would have 
```
%514 : Function = prim::Constant[name="AD_mm_backward_self"]()
grad_self.54 : Tensor = prim::CallFunction(%514, %868, %17) # <string>:204:28
```